README.md

```diff
@@ -120,10 +120,10 @@ And you can use IceRPC with a [DI container][icerpc-with-di]—or not. It's all
 ## Choose your IDL
 
 IceRPC provides a first-class byte-oriented API that allows you to make RPCs with the [IDL] and serialization format of
-your choice. For example, you can easily send Protobuf messages over IceRPC as illustrated by the [GreeterProtobuf]
-example.
+your choice.
 
-Another option—and the most common choice—is to use Slice to define the contract between your clients and servers.
+IceRPC comes with full support for two IDLs: Slice (described below) and [Protobuf]. You can select either Slice or
```

Comment:
You _use_ an IDL to define contracts, you don't _select_ an IDL to define contracts.
```suggestion
IceRPC comes with full support for two IDLs: Slice (described below) and [Protobuf]. You can use either Slice or
```

Or you could keep `select` and say:
> You can select either Slice or Protobuf for defining the contract ...